controllers/articlesController.js

```diff
@@ -3,9 +3,11 @@ const selectArticles = require("../models/articlesModel.js");
 const getArticles = (req,res)=>{
     selectArticles(req.query.sort_by,req.query.order,req.query.topic)
     .then((articlesData)=>{
-
         res.status(200).send({articles : articlesData});
     })
+    .catch((err)=>{
+        res.status(err.status).send({msg : err.msg})
+    })
```

Comment:
If you invoke `next` (and remember to pass is on line 3) you should be able to pass this directly to your error handling middleware in your app 👍